README.md

```diff
@@ -1,59 +1,228 @@
 
-[npm-url]:https://npmjs.org/package/@moxy/{package-name}
-[downloads-image]:https://img.shields.io/npm/dm/@moxy/{package-name}.svg
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-contentful-image
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-contentful-image.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-contentful-image.svg
+[build-status-url]:https://github.com/moxystudio/react-contentful-image/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-contentful-image/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-contentful-image
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-contentful-image/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-contentful-image
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-contentful-image.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-contentful-image?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-contentful-image.svg
 
-{package-description}
+A react image rendering using Contentful Images API.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-contentful-image
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+[Contentful](https://www.contentful.com/) provides a very powerful [Images API](https://www.contentful.com/developers/docs/references/images-api/) that besides retrieving image files in spaces, it provides manipulation features such as resizing, cropping and compressing.
+
+This react component will return a `<picture>` element. The `<picture>` contains at least one `<source>` element if no manipulations are being made and two `<source>` elements if there are manipulations. It also contains the native `<img>` element as fallback for browsers that do not support the `<picture>` element.
```

Comment:
```suggestion
This react component returns a `<picture>` element. If no manipulations are made, the `<picture>` contains at least one `<source>` element, otherwise it will have two `<source>` elements. It also contains the native `<img>` element as fallback for browsers that do not support the `<picture>` element.
```

Reply:
I did rephrase it a little bit more to avoid "element" or "elements" repetition.